File: Easy-Array/Maxout.java
// WAP to find the Largest element of Array
import java.util.Scanner;

class Maxout
{
   public static int Maximum(int arr[])
   {
       // Initialize maximum element
      int max=-1;
      // Traverse array elements
      for(int i=0;i<arr.length;i++)
      {
        // Check if current element is greater than max
        if(arr[i]>max)
        {
            // Update max
            max=arr[i];
        }
      }
      // Return max element of array
      return max;
   } 
   public static void main(String[] args)
   {
       Scanner sc=new Scanner(System.in);
       System.out.println("Size of Array");
       int n=sc.nextInt();
       int arr[]=new int[n];
       System.out.println("Enter the elements of array");
       for(int i=0;i<n;i++)
       {
        arr[i]=sc.nextInt();
        }
        System.out.println("Maximum element of array is "+Maximum(arr));
   }    
}

File: Easy-Array/Maxout.py
#  WAP to find the Largest element of Array


def find_largest(arr):
    if not arr:
        return None  # Return None for an empty array

    largest = arr[0]  # Initialize largest with the first element

    for num in arr:
        if num > largest:
            largest = num  # Update largest if current number is greater

    return largest

number_of_elements = int(input("Enter the number of elements in the array: "))
array = []
print("Enter the elements of the array:")
for _ in range(number_of_elements):
    element = int(input())
    array.append(element)
largest_element = find_largest(array)
if largest_element is not None:
    print(f"The largest element in the array is: {largest_element}")
else:
    print("The array is empty.")